Cargo.toml

```diff
@@ -51,6 +51,7 @@ tonic-build = { version = "0.3", features = ["prost"] }
 name = "integration"
 
 [dev-dependencies]
+serde = { version = "1.0", features = ["derive"] }
```

Comment:
No longer needed as it's fixed on `master`.

Reply:
At least switch the version to `1`, that diff should disappear on merge.

Reply:
fixed